cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/PreferredLeaderElectionGoalTest.java

```diff
@@ -108,9 +111,52 @@ public void testOptimizeWithDemotedBrokers() throws KafkaCruiseControlException
     }
   }
 
+  @Test
+  public void testOptimizeWithDemotedDisks() {
+    ClusterModel clusterModel = createClusterModel(true, true)._clusterModel;
```

Comment:
Do we also test with a cluster model having demoted (1) disks, and (2) brokers at the same time?